templates/resources/namespace_v1.md.tmpl

```diff
@@ -0,0 +1,127 @@
+---
+subcategory: "core/v1"
+page_title: "Kubernetes: kubernetes_namespace_v1"
+description: |-
+  Kubernetes supports multiple virtual clusters backed by the same physical cluster. These virtual clusters are called namespaces.
+---
+## {{ .Name }}
+
+{{ .Description }}
+
+{{ .SchemaMarkdown	}}
+
+## Example Usage
+
+{{tffile "examples/resources/namespace_v1/example_1.tf"}}
+
+## Argument Reference
```

Comment:
This part overlaps with what is rendered by `{{ .SchemaMarkdown }}`. Shell we remove it from here?